cli.js

```diff
@@ -17,4 +17,15 @@ var cli = meow({
 	string: ['_']
 });
 
-cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite});
+function errorHandler(err) {
+	if (err) {
+		console.error(err.message);
+		process.exit(1);
+	}
+}
+
+try {
+	cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite}, errorHandler);
+} catch (err) {
+	errorHandler(err);
```

Comment:
Do we really need the `errorHandler()` function if we aren't reusing it anywhere else? I'd rather do:

```js
} catch (err) {
    console.error(err.message);
    process.exit(1);
}